File: from-1201-to-1300/1233-remove-sub-folders-from-the-filesystem/solution.go
func removeSubfolders(folder []string) []string {
    sort.Strings(folder)
    result := []string{folder[0]}
    for i := 1; i < len(folder); i ++ {
        lastFolder := result[len(result) - 1] + "/"
        if !strings.HasPrefix(folder[i], lastFolder) {
            result = append(result, folder[i])
        }
    }
    return result
}

File: from-1201-to-1300/1233-remove-sub-folders-from-the-filesystem/solution.py
class Solution:
    def removeSubfolders(self, folder: List[str]) -> List[str]:
        folder.sort()
        result = [folder[0]]

        for i in range(1, len(folder)):
            # Get the last added folder path and add a trailing slash
            last_folder = result[-1] + '/'
            
            # Check if current folder starts with last_folder
            # If it doesn't start with last_folder, then it's not a subfolder
            if not folder[i].startswith(last_folder):
                result.append(folder[i])
        
        return result